src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -24,6 +24,23 @@ $(function () {
     render(URI(window.location.href));
   });
 
+  function updateDocString(node) {
+    if (node instanceof Array) {
+      for (var idx in node) {
+        updateDocString(node[idx]);
+      }
+    } else if (node instanceof Object) {
+      var docString = node['docString'];
+      if (docString && typeof docString === 'string') {
+        node['docString'] = docString.replace(/\n/gi, '<br>');
```

Comment:
Perhaps we should also add closing tags if there are any missing ones? Otherwise, the document layout will be broken badly. Alternatively, we could escape the special characters such as `<` and `>` into `&lt;` and `&gt;`?